lib/types/multipart.js

```diff
@@ -195,12 +195,17 @@ function Multipart (boy, cfg) {
 
         onData = function (data) {
           if ((nsize += data.length) > fileSizeLimit) {
-            const extralen = (fileSizeLimit - (nsize - data.length))
+            const extralen = nsize - fileSizeLimit
```

Comment:
can you explain your train of thought behind this change? This is clearly very different to the original math, but I assume there is method to it?

Reply:
Ok, this was quick math. But that should be fixed now